packages/site-client/src/components/wco-catalog-search.ts

```diff
@@ -4,28 +4,87 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
- import {html, css, LitElement} from 'lit';
- import {customElement} from 'lit/decorators.js';
- 
- @customElement('wco-catalog-search')
- export class WCOCatalogSearch extends LitElement {
-   static styles = css`
-     :host {
-       display: flex;
-       align-items: center;
-     }
-   `;
- 
-   render() {
-     return html`
-       Search: <input>
-     `;
-   }
- }
- 
- declare global {
-   interface HTMLElementTagNameMap {
-     'wco-catalog-search': WCOCatalogSearch;
-   }
- }
- 
\ No newline at end of file
+import {html, css, LitElement} from 'lit';
+import {customElement, query, state} from 'lit/decorators.js';
+import {gql, createClient, defaultExchanges} from '@urql/core';
```

Comment:
I thought we were using apollo, why are we also using urql?

Reply:
I wanted to try it out, and it's pure modules so doesn't require the common JS plug-in in web dev server.